lintcode/utils/utils.h

```diff
@@ -7,12 +7,16 @@
 
 #ifndef UTIL_H_
 #define UTIL_H_
-
+#ifdef __linux__

```

Comment:
Why do you want to define INT_MIN? It is defined in limits.h. Does it cause issues? If so, the right solution is probably using the exact constant explicitly to avoid confusion:
https://stackoverflow.com/questions/3233054/error-int32-max-was-not-declared-in-this-scope